appveyor/crowdinSync.py

```diff
@@ -0,0 +1,55 @@
+# based on file from https://github.com/jcsteh/osara
+
+import time
+import os
+import glob
+import sys
+import zipfile
+import shutil
+import subprocess
+import requests
+
+AUTH_TOKEN = os.getenv("crowdinAuthToken")
```

Comment:
In case there was a copy-paste error. Also prevents " " being considered a valid key
```suggestion
AUTH_TOKEN = os.getenv("crowdinAuthToken").strip()
```